### PR TITLE
Code Grooming

### PR DESCRIPTION
**Pull request by Jimgskoop**
Cleaned up some string constants, JSLint errors, YDoc comments, etc.

### CODE REVIEW

File: scripts/lu-controls/Button.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
  * @extends Abstract
- * @param {HTMLElement} element The HTML element surrounded by the control
- * @param {Object} settings Configuration properties for this instance
  * @requires ptclass, Abstract
  * @version 0.1
  */ 
 Button = Class.create( Abstract, ( function() {
 
+  // CONSTANTS
+  var CLICK = 'click',
```
Comment (Iheartweb) [inline scripts/lu-controls/Button.js:16]: our namiming for events has typically been [EVENTNAME]_EVENT.

for example:

var FLOORED_EVENT = 'floored',
    SELECTED_EVENT = 'selected';

File: scripts/lu-controls/Button.js [hunk 2]
```diff
@@ -154,17 +156,17 @@ Button = Class.create( Abstract, ( function() {
...
-        $element.removeClass( 'lu-disabled' );
+        $element.removeClass( LU_DISABLED );
       };
 
       // EVENT BINDINGS
       Button.on( settings.on, function( event ) {
         event.preventDefault();
 
-        _.info( 'click', action );
+        _.info( CLICK, action );
```
Comment (Iheartweb) [inline scripts/lu-controls/Button.js:169]: Let's clean out any extraneous msgs out of anything that is ready for release.

**Comment by Iheartweb** [comment]
Just a few notes - But overall - NICE!!!

File: test-server.js [hunk 3]
```diff
@@ -37,6 +37,8 @@ var nodeStatic = require( 'node-static' ),
 		browser: null
 	};
 
+console.log(__dirname);
```
Comment (Iheartweb) [inline test-server.js:40]: if we'd like to leave in consoles - let's also have a config to turn them off

File: scripts/lu-controls/Button.js (hunk 1 above)
Comment (Jimgskoop) [inline scripts/lu-controls/Button.js:16]: fixed

File: scripts/lu-controls/Button.js (hunk 2 above)
Comment (Jimgskoop) [inline scripts/lu-controls/Button.js:169]: Fixed

File: test-server.js (hunk 3 above)
Comment (Jimgskoop) [inline test-server.js:40]: This is in test-server.js -- there are a LOT of other console statements there as well that are used for output within the Terminal.

**Pull request merged**